pontoon/contributors/templates/contributors/settings.html

```diff
@@ -31,14 +31,14 @@ <h2 id="username">{{ user.first_name }}</h2>
     {% endif %}
   </ul>
 
-  <div id="heading" class="clearfix">
+  <div id="user-preference" class="clearfix">
     <div id="homepage">
       <span class="label">Select your homepage</span>
       <span class="homepage">{% include 'teams/widgets/team_selector.html' %}</span>
     </div>
     <div id="preferred-locale">
-      <span class="label preferred-locale">Select preferred source locale</span>
-      <span class="preferred-locale">{% include 'teams/widgets/preferred_locale_selector.html' %}</span>
+      <span class="label">Select preferred source locale</span>
+      <span class='preferred-locale'>{% include 'teams/widgets/preferred_locale_selector.html' %}</span>
```

Comment:
Nit: please use double quotes.

Reply:
I actually removed that class in the refactor of team selector (https://github.com/mozilla/pontoon/pull/1474/commits/c9b672fe23836c08aaec3c0cbffd12161946f880) as it was no longer needed but will note the need for double quotes on classes.

Reply:
Whoops, false alarm. Sorry!